usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -7,10 +7,22 @@
 @pytest.fixture()
 def check_cloud_register(host):
     def f():
+        # There was an API change in registerutils and we have to check
+        # which version is in place on the system
+        deleted_interface_import = \
+            'from cloudregister.registerutils import check_registration'
+        result = host.run(
+            "sudo python3 -c '{0}'".format(deleted_interface_import)
+        )
+        # Old interface of is_registered()
+        is_registered_arg = 'registerutils.get_current_smt()'
+        if result.stdout.strip() == '1':
```

Comment:
This needs to check for the rc as stdout is expected to be empty unless there's an exception dumped:

```
if result.rc == 1:
```